Changed "Library Code" to "Ignore List" (#2382)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/javascript/reference.md

<!-- ====================================================================== -->
## Ignore a script or pattern of scripts

> Mark a script as belonging to the **Ignore List** when you want to ignore that script while debugging.  When marked as **Ignore List**, a script is obscured in the **Call Stack** pane, and you never step into the functions of the script when you step through your code.

Author: ```suggestion
Add a script to the **Ignore List** when you want to ignore that script while debugging.  When included in the **Ignore List**, a script is obscured in the **Call Stack** pane, and you never step into the functions of the script when you step through your code.
```

Author: fixed locally

> For example, in the following code snippet, line `A` uses `lib`, which is a third-party library.  If you are confident that the problem you are debugging isn't related to that third-party library, then it makes sense to mark the script as **Ignore List**.

Author: ```suggestion
For example, in the following code snippet, line `A` uses `lib`, which is a third-party library.  If you are confident that the problem you are debugging isn't related to that third-party library, then it makes sense to add the script to the **Ignore List**.
```

Author: done locally

}
```

### Mark a script as **Ignore List** from the Editor pane

To mark a script as **Ignore List** from the **Editor** pane:

1. Open the file.

1. Right-click anywhere in the file and then select **Add script to ignore list**:

   ![Marking a script as Ignore List from the Editor pane](../media/javascript-glitch-debug-js-sources-get-started-inputs-are-empty-editor-mark-as-library-code.msft.png)

### Mark a script as Ignore List from the Call Stack pane

To mark a script as **Ignore List** from the **Call Stack** pane:

*  Right-click a function in the script and then select **Add script to ignore list**:

>    ![Marking a script as Ignore List from the Call Stack pane](../media/javascript-glitch-debug-js-sources-get-started-inputs-are-empty-call-stack-mark-as-library-code.msft.png)

Author: ```suggestion
   ![Adding a script to the Ignore List from the Call Stack pane](../media/javascript-glitch-debug-js-sources-get-started-inputs-are-empty-call-stack-mark-as-library-code.msft.png)
```

Author: done locally

> ### Mark a script as Ignore List from Settings

Author: ```suggestion
### Add a script to the Ignore List from Settings
```

Author: done locally

To mark a single script or pattern of scripts from **Settings**:

1. Open [Settings](../customize/index.md).

1. Navigate to the **Ignore List** setting.

1. Click **Add pattern**.

> 1. Enter the script name or a regex pattern of script names to mark as **Ignore List**.

Author: ```suggestion
1. Enter the script name or a regex pattern of script names to add to the **Ignore List**.
```

Author: done locally

1. Click **Add**.

>    ![Marking a script as Ignore List from Settings](../media/javascript-framework-library-code.msft.png)

Author: done locally

>    ![Marking a script as Ignore List from Settings](../media/javascript-framework-library-code.msft.png)

Author: done locally

}
```

> #### Mark a script as **Ignore List** from the Editor pane

Author: fixed locally

> To mark a script as **Ignore List** from the **Editor** pane:

Author: fixed locally

1. Open the file.

1. Right-click anywhere in the file and then select **Add script to ignore list**:

>    ![Marking a script as Ignore List from the Editor pane](../media/javascript-glitch-debug-js-sources-get-started-inputs-are-empty-editor-mark-as-library-code.msft.png)

Author: fixed locally

> #### Mark a script as Ignore List from the Call Stack pane

Author: fixed locally

> To mark a script as **Ignore List** from the **Call Stack** pane:

Author: fixed locally
